functest/create_member_test.go

```diff
@@ -47,3 +47,18 @@ func TestCreateMemberWithBadKey(t *testing.T) {
 	require.NotNil(t, err)
 	require.Contains(t, err.Error(), fmt.Sprintf("problems with decoding. Key - %s", member.pubKey))
 }
+
+func TestCreateMembersWithSameName(t *testing.T) {
+	member, err := newUserWithKeys()
+	require.NoError(t, err)
+	member.ref = root.ref
+
+	addBurnAddresses(t)
+
+	_, err = retryableCreateMember(member, "contract.createMember", map[string]interface{}{}, false)
```

Comment:
Last param in `retryableCreateMember` must be `true` (we want new valid publicKey for every retry attempt)